### PR TITLE
Migrate NVText Normalizing APIs to Pylibcudf

### PR DESCRIPTION
**Pull request by Matt711**
## Description

Apart of #15162.
## Checklist
- [x] I am familiar with the [Contributing Guidelines](https://github.com/rapidsai/cudf/blob/HEAD/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.

### CODE REVIEW

File: python/pylibcudf/pylibcudf/tests/test_nvtext_normalize.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+@pytest.fixture(scope="module")
+def input_data():
+    arr1 = ["éâîô\teaio", "ĂĆĖÑÜ", "ACENU", "$24.08", "[a,bb]"]
+    arr2 = ["a b", "  c  d\n", "e \t f "]
+    return pa.array(arr1), pa.array(arr2)
+
+
+def test_normalize_spaces(input_data):
+    _, input_col = input_data
```
Comment (bdice) [inline python/pylibcudf/pylibcudf/tests/test_nvtext_normalize.py:17]: This is a weird pattern to have two data inputs and only use one of them. Can we make two fixtures instead?

**Review by bdice — CHANGES_REQUESTED** [review]
Just one blocking request on the tests.

**Review by bdice — APPROVED** [review]
Please write a PR description before merging!

**Comment by Matt711** [comment]
> Please write a PR description before merging!

Thanks!

**Pull request merged**